alexandria/core/models.py

```diff
@@ -246,6 +246,11 @@ def set_checksum(self):
     def set_content_vector(self):
         self.content.file.file.seek(0)
         parsed_content = tika.parser.from_buffer(self.content.file.file)
+
+        if not parsed_content["content"]:
+            self.content_vector = SearchVector(Value(""))
```

Comment:
Should a file with no content be searchable (through name)?

Reply:
Only if the name is part of the searchable space in the "success case" as well. Otherwise, I'd leave it to the filters to find files/documents by name

Reply:
The name is also added with more weight to the searchables in the normal case.
Added the name to the content_vector even when content is None